tabbycat/tournaments/utils.py

```diff
@@ -174,3 +182,41 @@ def _wrapped(tournament):
 neg_abbr = _get_side_name('neg_abbr')
 aff_team = _get_side_name('aff_team')
 neg_team = _get_side_name('neg_team')
+
+
+def send_standings_emails(tournament, teams, request):
+    messages = []
+
+    subject_temp = Template(tournament.pref('team_points_email_subject'))
+    message_temp = Template(tournament.pref('team_points_email_message'))
+
+    context = {'TOURN': str(tournament)}
+
+    message_link = ''
+    if tournament.pref('public_team_standings'):
+        url = request.build_absolute_uri(reverse_tournament('standings-public-teams-current', tournament))
+        message_link += "\n\n" + tournament.pref('team_points_email_link_text') + "\n" + url
+
+    for team in teams:
+        context['POINTS'] = str(team.points_count)
+        context['TEAM'] = team.short_name
+
+        subject = subject_temp.render(Context(context))
+
+        for speaker in team.speakers:
+            if speaker.email is None:
+                continue
+
+            context['USER'] = str(speaker)
```

Comment:
Probably better to use `speaker.name`, since that's what we want, and the behavior of `str(speaker)` (which you should think of as "a sensible string representation of the speaker" rather than "the speaker's name") is liable to change. 🙂